terraform/modules/matrix-service/infra/converter_batch.tf

```diff
@@ -0,0 +1,233 @@
+    instance_type = [
+      "m4"
+    ]
+    image_id = "${var.converter_cluster_ami_id}"
+    subnets = ["${data.aws_subnet_ids.matrix_vpc.ids}"]
+    security_group_ids = [
+      "${aws_vpc.vpc.default_security_group_id}"
+    ]
+    ec2_key_pair = "matrix-service-${var.deployment_stage}"
+    instance_role = "${aws_iam_instance_profile.ecsInstanceRole.arn}"
+  }
+  depends_on = [
+    "aws_iam_role_policy_attachment.AWSBatchServiceRole"
+  ]
+}
+
+resource "aws_batch_job_queue" "converter_job_queue" {
+  name = "dcp-matrix-converter-queue-${var.deployment_stage}"
+  compute_environments = [
+    "${aws_batch_compute_environment.converter_compute_env.arn}"]
+  priority = 1
+  state = "ENABLED"
+}
+
+resource "aws_iam_policy" "converter_job_policy" {
+  name = "dcp-matrix-converter-job-${var.deployment_stage}"
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "s3",
+        "Effect": "Allow",
+        "Action": [
+          "s3:*"
+        ],
+        "Resource": [
+          "arn:aws:s3:::dcp-matrix-service-results-${var.deployment_stage}",
+          "arn:aws:s3:::dcp-matrix-service-results-${var.deployment_stage}/*"
+        ]
+      },
+      {
+        "Sid": "DynamoPolicy",
+        "Effect": "Allow",
+        "Action": [
+          "dynamodb:UpdateItem",
+          "dynamodb:GetItem"
+        ],
+        "Resource": [
+          "arn:aws:dynamodb:${var.aws_region}:${var.account_id}:table/dcp-matrix-service-state-table-${var.deployment_stage}"
+        ]
+      },
+      {
+        "Sid": "S3ReadFors3fs",
+        "Effect": "Allow",
+        "Action": [
+            "s3:ListAllMyBuckets",
+            "s3:HeadBucket"
+        ],
+        "Resource": "*"
+      }
+    ]
+}
+POLICY
+}
+
+resource "aws_iam_role" "converter_job_role" {
+  name = "dcp-matrix-converter-job-${var.deployment_stage}"
+  assume_role_policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ecs-tasks.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        },
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "lambda.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+POLICY
+}
+
+resource "aws_iam_role_policy_attachment" "converter_job_role" {
+  role = "${aws_iam_role.converter_job_role.name}"
+  policy_arn = "${aws_iam_policy.converter_job_policy.arn}"
+}
+
+resource "aws_batch_job_definition" "converter_job_def" {
+    name = "dcp-matrix-converter-job-definition-${var.deployment_stage}"
+    type = "container"
+    container_properties = <<CONTAINER_PROPERTIES
+{
+  "command": [],
+  "image": "humancellatlas/matrix-converter:1",
+  "memory": 4096,
+  "vcpus": 4,
```

Comment:
Is there a reason we're setting this lower than the min vCPUs on the batch compute environment?